packages/@aws-cdk/aws-ecs/lib/external/external-task-definition.ts

```diff
@@ -59,20 +65,27 @@ export class ExternalTaskDefinition extends TaskDefinition implements IExternalT
     return new ImportedTaskDefinition(scope, id, {
       taskDefinitionArn: attrs.taskDefinitionArn,
       compatibility: Compatibility.EXTERNAL,
-      networkMode: NetworkMode.BRIDGE,
+      networkMode: attrs.networkMode,
       taskRole: attrs.taskRole,
     });
   }
 
+  /**
+   * The networking mode to use for the containers in the task.
+   */
+  public readonly networkMode: NetworkMode;
+
   /**
    * Constructs a new instance of the ExternalTaskDefinition class.
    */
   constructor(scope: Construct, id: string, props: ExternalTaskDefinitionProps = {}) {
     super(scope, id, {
       ...props,
       compatibility: Compatibility.EXTERNAL,
-      networkMode: NetworkMode.BRIDGE,
+      networkMode: (props.networkMode ?? NetworkMode.BRIDGE),
     });
+
+    this.networkMode = props.networkMode ?? NetworkMode.BRIDGE;
```

Comment:
```suggestion
```

Reply:
This is already done by passing in `props.networkMode ?? NetworkMode.BRIDGE` to the super constructor.